openwisp_monitoring/monitoring/static/monitoring/js/chart.js

```diff
@@ -84,15 +95,21 @@
             label = data.traces[i][0].replace(/_/g, ' ');
 
             if (data.summary_labels){
-              summaryLabels.push([key, data.summary_labels[i]]);
+                if (data.trace_order) {
+                    summaryLabels.push([key, data.summary_labels[data.trace_order.indexOf(key)]]);
+                }
+                else {
+                  summaryLabels.push([key, data.summary_labels[i]]);
+                }
             }
             var options = {
                     name: label,
                     type: type,
                     mode: mode,
                     fill: data.fill || 'tozeroy',
                     hovertemplate: [],
-                    y: []
+                    y: [],
+                    _key: key,
```

Comment:
This part is a bit obscure, the options object is used to control plotly features, but this line doesn't seem related to plotly but more like an internal thing, can you please clarify this with a comment?